websites/C/Custom Status/dist/metadata.json

```diff
@@ -6,7 +6,7 @@
 	},
 	"service": "Custom Status",
 	"description": {
-		"en": "Custom Status is a way to set custom statuses using PreMiD. You can customize everything but using custom images. You have to visit [this page](https://eggsy.xyz/projects/premid/custom-status) to start using!",
+		"en": "Custom Status is a way to set custom statuses using PreMiD. You can customize everything except the name. You have to visit [this page](https://eggsy.xyz/projects/premid/custom-status) to start using it!",
 		"tr": "Custom Status, Discord'un kendi özelliği gibi isteğinize göre ayarlanabilen bir oynuyor kısmıdır. Bu servis ile PreMiD kullanarak istediğiniz resmi seçme olanağı dışında bir RPC ile yapabileceğiniz her şeyi yapabilirsiniz. Servisin çalışması için servisi ekleyip, [buradaki](https://eggsy.xyz/projects/premid/custom-status) sayfaya gitmelisiniz.",
 		"nl": "Custom Status staat je toe aangepaste statussen weer te geven met PreMiD. Je kunt alles aanpassen behalve het gebruik van eigen afbeeldingen. Bezoek [deze pagina](https://eggsy.xyz/projects/premid/custom-status) om te beginnen!",
```

Comment:
@QkeleQ10 or @Bas950 can you translate the changed string?

Reply:
```suggestion
		"nl": "Custom Status staat je toe aangepaste statussen weer te geven met PreMiD. Je kunt alles aanpassen behalve de naam. Bezoek [deze pagina](https://eggsy.xyz/projects/premid/custom-status) om te beginnen!",
```